java/pom.xml

```diff
@@ -63,7 +63,7 @@
 
     <properties>
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-      <grpc.version>1.0.0</grpc.version>
+      <grpc.version>1.0.2</grpc.version>
```

Comment:
@alainjobart recently pushed this to 1.0.4 in go-land -- should we match here? Maybe we can do it in the other PR if you want